openslides_backend/action/action.py

```diff
@@ -194,9 +191,6 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
                 ):
                     return
                 raise MissingPermission(self.permission)
-        else:
```

Comment:
Can the `if` be removed where this `else` belongs to? What does happen, if `self.permission` is not defined otherwise? If this may happen I would suggest using a `NotImplementedError` to catch errors, where a developer did not provide a permission and neither overwrote the `check_permissions` method.

Reply:
Well, in that case it would just throw a 403 on every call, which would be okay I guess (and would happen during tests, too, so the developer would catch it). But we can also change it if you want

Reply:
It will throw an PermissionDenied, so the developer will notice it.